### PR TITLE
feat: show code signin page with password disabled

### PR DESCRIPTION
**Pull request by notdu**
Current code not support show sign in form (application enabled code sign in) when password is disable.
It should work independent of each other

### CODE REVIEW

**Comment by hsluoyz** [comment]
@notdu demo image and video?

**Comment by notdu** [comment]
> 

@hsluoyz [here](https://drive.google.com/file/d/1urXCXGWCBs3B6ne7KOMmbfjLmNMJxPH5/view?usp=sharing) is the demo

**Comment by hsluoyz** [comment]
@notdu the UI is not correct, it still shows the "Password" box. And the tabs can be removed

image

**Comment by notdu** [comment]
@hsluoyz [updated](https://drive.google.com/file/d/19IYPyalZQQXsWnO8VyvHp4VykfFZJpim/view?usp=sharing), please review

File: web/src/auth/LoginPage.js [hunk 1]
```diff
@@ -515,7 +531,7 @@ class LoginPage extends React.Component {
                   id="input"
                   prefix={<UserOutlined className="site-form-item-icon" />}
                   placeholder={(this.state.loginMethod === "verificationCode") ? i18next.t("login:Email or phone") : i18next.t("login:username, Email or phone")}
-                  disabled={!application.enablePassword}
+                  disabled={!showForm}
```
Comment (hsluoyz) [inline web/src/auth/LoginPage.js:534]: Isn't `!showForm` always false?

File: web/src/auth/LoginPage.js [hunk 2]
```diff
@@ -530,7 +546,7 @@ class LoginPage extends React.Component {
           </Row>
           <div style={{display: "inline-flex", justifyContent: "space-between", width: "320px", marginBottom: AgreementModal.isAgreementRequired(application) ? "5px" : "25px"}}>
             <Form.Item name="autoSignin" valuePropName="checked" noStyle>
-              <Checkbox style={{float: "left"}} disabled={!application.enablePassword}>
+              <Checkbox style={{float: "left"}} disabled={!showForm}>
```
Comment (hsluoyz) [inline web/src/auth/LoginPage.js:549]: Isn't `!showForm` always false?

File: web/src/auth/LoginPage.js (hunk 1 above)
Comment (notdu) [reply web/src/auth/LoginPage.js:534]: no, if we turn off all 3 login method, it'll be true, and can auto signin

Comment (hsluoyz) [reply web/src/auth/LoginPage.js:534]: @notdu am I understanding right?

image

Comment (notdu) [reply web/src/auth/LoginPage.js:534]: yes, you are right, i think we do not need `disable` attribute. I'll remove it

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**